packages/fury-adapter-oas3-parser/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Fury OAS3 Parser Changelog
 
-## Master
+## 0.6.0 (26-02-19)
```

Comment:
One thing I noticed here is this changelog uses a slightly different date format, I'll fix in separate PR (don't want to mix releasing with other changes).